code/modules/mob/mob_helpers.dm

```diff
@@ -34,12 +34,18 @@
  * probability controls the chance it chooses the passed in zone, or another random zone
  * defaults to 80
  */
-/proc/ran_zone(zone, probability = 80)
-	if(prob(probability))
-		zone = check_zone(zone)
-	else
-		zone = pickweight(list(BODY_ZONE_HEAD = 1, BODY_ZONE_CHEST = 1, BODY_ZONE_L_ARM = 4, BODY_ZONE_R_ARM = 4, BODY_ZONE_L_LEG = 4, BODY_ZONE_R_LEG = 4))
-	return zone
+/proc/ran_zone(zone = FALSE, probability = 80)
+	if(zone && prob(probability))
```

Comment:
This is changing targeting pretty significantly without passed in zone since beforehand it was 80% for body ? Was this function used anywhere else without the argument?

Reply:
It's been two months but I don't think so. It's not used in very many places.